configs/experiment/im2im/segmentation_plugin.yaml

```diff
@@ -26,24 +31,21 @@ exclude_mask_col: exclude_mask
 base_image_col: base_image
 
 # data params
-spatial_dims: 3
+spatial_dims: MUST_OVERRIDE # int value, req for first training, should not change after
 input_channel: 0
 raw_im_channels: 1
 
 trainer:
-  max_epochs: 100
+  max_epochs: 1 # must override for training
```

Comment:
Yeah, it's not a perfect system for a few reasons, principally because we are using the same config for training/prediction, but training and prediction have slightly different requirements for what must be overridden. I'd be happy to discuss this more, but I want to prioritize getting this merged for Benji's sake.